test/unit/org/apache/cassandra/cql3/CQLTester.java

```diff
@@ -448,51 +446,16 @@ public void afterTest() throws Throwable
         aggregates = null;
         user = null;
 
-        // We want to clean up after the test, but dropping a table is rather long so just do that asynchronously
-        schemaCleanup.execute(() -> {
-            try
-            {
-                logger.debug("Dropping {} materialized view created in previous test", viewsToDrop.size());
-                for (int i = viewsToDrop.size() - 1; i >= 0; i--)
-                    schemaChange(String.format("DROP MATERIALIZED VIEW IF EXISTS %s.%s", KEYSPACE, viewsToDrop.get(i)));
-
-                for (int i = tablesToDrop.size() - 1; i >= 0; i--)
-                    schemaChange(String.format("DROP TABLE IF EXISTS %s.%s", KEYSPACE, tablesToDrop.get(i)));
-
-                for (int i = aggregatesToDrop.size() - 1; i >= 0; i--)
-                    schemaChange(String.format("DROP AGGREGATE IF EXISTS %s", aggregatesToDrop.get(i)));
-
-                for (int i = functionsToDrop.size() - 1; i >= 0; i--)
-                    schemaChange(String.format("DROP FUNCTION IF EXISTS %s", functionsToDrop.get(i)));
-
-                for (int i = typesToDrop.size() - 1; i >= 0; i--)
-                    schemaChange(String.format("DROP TYPE IF EXISTS %s.%s", KEYSPACE, typesToDrop.get(i)));
-
-                for (int i = keyspacesToDrop.size() - 1; i >= 0; i--)
-                    schemaChange(String.format("DROP KEYSPACE IF EXISTS %s", keyspacesToDrop.get(i)));
-
-                // Dropping doesn't delete the sstables. It's not a huge deal but it's cleaner to cleanup after us
-                // Thas said, we shouldn't delete blindly before the TransactionLogs.SSTableTidier for the table we drop
-                // have run or they will be unhappy. Since those taks are scheduled on StorageService.tasks and that's
-                // mono-threaded, just push a task on the queue to find when it's empty. No perfect but good enough.
-
-                final CountDownLatch latch = new CountDownLatch(1);
-                ScheduledExecutors.nonPeriodicTasks.execute(new Runnable()
-                {
-                    public void run()
-                    {
-                        latch.countDown();
-                    }
-                });
-                latch.await(2, TimeUnit.SECONDS);
-
-                removeAllSSTables(KEYSPACE, tablesToDrop);
-            }
-            catch (Exception e)
-            {
-                throw new RuntimeException(e);
-            }
-        });
+        try
+        {
+            Schema.instance.transform(schema -> schema.without(List.of(KEYSPACE_PER_TEST))
```

Comment:
what is the consequence of that change wrt sstable deletion?

Reply:
I'm not sure I understand the question, could you elaborate what are you worrient about?

Reply:
I don't know, I believe this is not needed anymore.
- schema changes are synchronous
- dropping a table follows dropping all the sstables (which is synchronous as well)
(see that previously it could lead to more problems, deleting tables and sstables were executed async in the after-each block and could span the execution of the next test method)